components/Book/index.js

```diff
@@ -8,14 +8,12 @@ export default function Book({ book }) {
   return (
     <StyledFrame>
       <div>
-        <StyledImage>
-          <Image
-            src={imageURL}
-            alt={title}
-            width={dimensions.width}
-            height={dimensions.height}
-          />
-        </StyledImage>
+        <StyledImage
+          src={imageURL}
+          alt={title}
+          width={dimensions.width}
+          height={dimensions.height}
+        ></StyledImage>
       </div>
```

Comment:
For some strange reasons I can't leave a suggestion. 😫

But here are some improvements I would like to share with you:
- I think it would make sense to rename the Book component to BookImage 
- you could remove the div-element
- you the `StyledImage` component as self-closing tag
- you could remove the` import Image from "next/image";` because you do not use the image component here